packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-listbox-multi.tsx

```diff
@@ -240,9 +241,27 @@ export class GuxListboxMulti {
 
   // get list of listbox option elements
   private setListboxOptions(): void {
-    this.listboxOptions = (
-      Array.from(this.root.children) as HTMLGuxOptionMultiElement[]
-    ).filter(element => element.tagName === 'GUX-OPTION-MULTI');
+    const options: HTMLGuxOptionMultiElement[] = [];
+    const listChildren = Array.from(this.root.children);
+    listChildren.map(child => {
+      if (isOptionGroup(child)) {
+        const childOptions = Array.from(
+          child.children
+        ) as HTMLGuxOptionMultiElement[];
+        return options.push(...childOptions);
+      }
+      return options.push(child as HTMLGuxOptionMultiElement);
+    });
+
+    const filteredOptions: HTMLGuxOptionMultiElement[] = options.filter(
+      element => element.tagName === 'GUX-OPTION-MULTI'
+    );
+    this.listboxOptions = [...filteredOptions];
+
+    const lastChild = listChildren[listChildren.length - 1];
+    if (lastChild && isOptionGroup(lastChild))
+      lastChild.setAttribute('divider', 'false');
```

Comment:
We should see is it possible to do this with `:has-context` (https://developer.mozilla.org/en-US/docs/Web/CSS/:host-context) instead of in javascript.

Also I would use `lastChild.divider = false` instead of setting it via an attribute.